pyproject.toml

```
@@ -29,7 +29,7 @@ sacrebleu = {extras = ["ja"], version = "^2.4.1"}
 jiwer = "^3.0.4"
 openai = "^1.52.2"
 google-api-python-client = "^2.131.0"
-vllm = {version = "^0.5.5", optional = true }
+vllm = {version = "^0.6.4", optional = true }
```

```suggestion
vllm = {version = "^0.6.4.post1", optional = true }
```
I think it is better to specify `0.6.4.post1` which is bug fix version
https://github.com/vllm-project/vllm/releases